railties/test/application/configuration_test.rb

```diff
@@ -1821,6 +1821,27 @@ class D < C
       assert_equal "dev_db",  ar_config["development"]["database"]
     end
 
+    test "database.yml can include other yaml files for configuration" do
```

Comment:
I think this test is too specific. We just want to make sure `__FILE__` returns the name of the file and not `(erb)`.

```ruby
      app_file "config/database.yml", <<~YAML
        development:
          username: bobby
          adapter: sqlite3
          database: 'dev_db'
          some_path: <%= __FILE__ %>
      YAML

  assert_equal file_path, ar_config["development"]["some_path"]
```